functests/config.go

```diff
@@ -22,11 +22,11 @@ var OcsSubscriptionChannel string
 // UpgradeFromOcsSubscriptionChannel is the name of the ocs subscription channel to upgrade from
 var UpgradeFromOcsSubscriptionChannel string
 
-// OcsRegistryImage is the ocs-registry container image to use in the deployment
-var OcsRegistryImage string
+// OcsCatalogImage is the ocs catalog container image to use in the deployment
```

Comment:
Just a small suggestion, what do you think of instead saying "the OCS CatalogSource container image"? This is just a nitpick to be more technically explicit about what this image is actually used for/by.